docs/fast_data/inputs_and_outputs.md

```diff
@@ -185,46 +210,57 @@ Delete operation:
 ```yaml
 asyncapi: 2.6.0
 info:
-  title: Debezium Data Change Producer
+  title: Data Change API
   version: 1.0.0
 channels:
-  DebeziumDataChangeChannel:
+  Debezium:
     publish:
       message:
-        name: debezium data change
+        name: Debezium data change message
         payload:
           type: object
-          additionalProperties: false
+          required:
+            - key
+            - value
+            - timestamp
+            - offset
           properties:
-            key: {}
+            key: 
+              type: object
+              description: Record's primary keys
+              additionalProperties: true
             value:
               type: object
-              additionalProperties: false
+              required:
+                - op
+                - source
               properties:
                 op:
                   type: string
+                  description: Operation type. Can be *create/snapshot(r)*, *update* or *delete*
                   enum:
                     - c
+                    - r
                     - u
                     - d
                 before:
```

Comment:
(t)runcate messages? (despite we do not suppor them
```suggestion
                  description: Operation type. Can be *create/snapshot(r)*, *update*, *truncate* (not supported by Fast Data) or *delete*
                  enum:
                    - c
                    - r
                    - u
                    - d
                    - t
                before:
```)

Reply:
I didn't put it on purpose so the user understands that if he wants to drop a whole table he needs to do it by deleting each record one by one, otherwise the RTU won't do anything

Reply:
Smart. I'm still unsure about it but it seems reasonable since we're explaining what we are expecting from the user.